bin/varnishd/cache/cache_hash.c

```diff
@@ -355,6 +355,42 @@ hsh_insert_busyobj(const struct worker *wrk, struct objhead *oh)
 	return (oc);
 }
 
+/*---------------------------------------------------------------------
+*/
+
+static unsigned
+hsh_rush_match(struct req *req)
```

Comment:
Would `hsh_match_vary()` be a better name for this function? It seems it is not solely concerned with the rushing, but more with the vary matching.

Reply:
Well, it is concerned about matching a rushed objcore with a request, that's why it fails to match if there is no rushed objcore.

I guess the naming convention would rather be `hsh_vary_match()`, as a "shorthand" for `VRY_Match()`. But it's not limited to variants, the object must be cacheable in the first place to match.

Not sure how I feel about this one, I think `hsh_rush_match()` is more accurate.